src/templates/viewer-spec.yaml.j2

```
@@ -0,0 +1,38 @@
+# Source: manifests/apps/volumes-web-app/upstream/base/viewer-spec.yaml
+# Note: the volumes-web-app allows expanding strings using ${VAR_NAME}
+# You may use any environment variable. This lets us e.g. specify images that can be modified using kustomize's image transformer.
+# Additionally, 'PVC_NAME', 'NAME' and 'NAMESPACE' are defined
+# Name of the pvc is set by the volumes web app
+pvc: $NAME
+podTemplate:
+  containers:
+    - name: main
+      image: $VOLUME_VIEWER_IMAGE
```

I guess we want the value from of `volume-viewer-image` from the charm config, so this would become something like
```suggestion
      image: {{ volume_viewer_image }}
```

And then in the charm code we would have to pass `volume_viewer_image` as context for rendering this file.

---

This env variable is defined [here](https://github.com/canonical/kubeflow-volumes-operator/blob/c438d370d6c1671ef4772cf94ad20a50bc4e6eda/src/charm.py#L165) so we should be good to go